include/acquire.zarr.h

```diff
@@ -0,0 +1,274 @@
+#ifndef H_ACQUIRE_ZARR_V0
+#define H_ACQUIRE_ZARR_V0
+
+#include "zarr.types.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    typedef struct ZarrStreamSettings_s ZarrStreamSettings;
+    typedef struct ZarrStream_s ZarrStream;
+
+    /**
+     * @brief Get the version of the Zarr API.
+     * @return The version of the Zarr API.
+     */
+    uint32_t Zarr_get_api_version();
+
+    /**
+     * @brief Set the log level for the Zarr API.
+     * @param level The log level.
+     * @return ZarrStatus_Success on success, or an error code on failure.
+     */
+    ZarrStatus Zarr_set_log_level(ZarrLogLevel level);
```

Comment:
Yeah, it's the former. I like it here because users can still set to debug and apply any additional filters they like to the log, or they can use the built-in filters. It's not called out in the doc (I can change that), but in the implementation all calculation is skipped for lower-level logs, and this can be a performance improvement.